pkg/kubelet/cri/remote/conversion.go

```diff
@@ -0,0 +1,322 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package remote
+
+import (
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
+	"k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+func fromV1alpha2VersionResponse(from *v1alpha2.VersionResponse) *runtimeapi.VersionResponse {
+	if from == nil {
+		return nil
+	}
+	data, _ := from.Marshal()
+	resp := &runtimeapi.VersionResponse{}
+	resp.Unmarshal(data)
```

Comment:
Do we need to do a full on conversion with marshal and unmarshalling? I think I suggested on the previous PR that it would be safe and faster to do an unsafe memory cast instead since we know the types are byte for byte identical and can catch any issues with a unit test, and hence avoid the overhead of this.

Reply:
Right, changed to a plain memory conversion. :+1: